packages/common/container-definitions/src/loader.ts

```diff
@@ -489,15 +485,11 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
 	/**
 	 * Exposes the entryPoint for the container.
 	 * Use this as the primary way of getting access to the user-defined logic within the container.
-	 * If the method is undefined or the returned promise returns undefined (meaning that exposing the entryPoint
-	 * hasn't been implemented in a particular scenario) fall back to the current approach of requesting the default
-	 * object of the container through the request pattern.
 	 *
 	 * @remarks The plan is that eventually IContainer will no longer implement IFluidRouter (and thus won't have a
-	 * request() method), this method will no longer be optional, and it will become the only way to access
-	 * the entryPoint for the container.
+	 * request() method) and this method will become the only way to access the entryPoint for the container.
```

Comment:
We should remember to remove this conmment once request is removed.

Reply:
Actually, maybe we can remove this comment now. Entry point is required and it doesn't matter to it if request exists.